source/shared/cpp/ObjectModel/HostOptions.h

```diff
@@ -52,11 +52,11 @@ struct SeparationOptions
 {
     unsigned int spacing = 10;
     unsigned int lineThickness = 0;
-    std::string lineColor;
+    std::string lineColor = "#FF101010";
 };
 struct ImageSizeOptions
 {
-    unsigned int smallSize = 60;
+    unsigned int smallSize = 80;
```

Comment:
>unsigned int smallSize = 80; [](start = 4, length = 28)

Changing this will affect the UWP renderer as well. If sizes are overall smaller in android, does it make sense to add an offset during image rendering directly rather than modifying the default host config?